### ISSUE TITLE
Cannot render font awesome icons

### ISSUE THREAD

**Issue by abhi1693** [octobercms/october#2434]
##### Expected behavior
Font awesome icons should render after a cache clean.

##### Actual behavior
Font awesome icons are not being rendered.

##### Reproduce steps
I have read the documentation provided on your website for implementing the Font awesome icons and I have done exactly that. But for some reasons I'm unable to view the font awesome icons.

Code for font awesome written on my website:
```
<a href="#" class="author-social-link small" target="_blank">
<i class="fa icon-facebook-square icon-lg"></i>
</a>
```

Output: 
![fa](https://cloud.githubusercontent.com/assets/5083532/19541448/0edb965e-9684-11e6-959a-aa7c35a2b90f.png)

The icons should show up inside the box next to my name ie `Abhimanyu Saharan`. But nothing is being rendered.

##### October build
365

**Comment by Patroklo** [comment]
Please, use your firebug or simmilar tool to check what's there. It could be that you are not loading the `storm.css` file, so there won't be any font awesome icon in your web, or it could be that you are overriding the class with another css, for example. 

You can also have a problem with css from another section, for example it could be that your `a` its so small that you can't see its content.

Also, the `fa` it's not really neccessary, I have checked your code with and without it and it works both ways.

**Comment by abhi1693** [comment]
@Patroklo My website was loading all the files that October CMS requested. I have not changed the css structure of the core files. But, I tried to find out the class that is, `icon-facebook-sqaure` but I could not find it anywhere in the files that came along with the installation. 

So, I tried to install October via github and everything was working fine. So, why is it that the installation via github worked and but via a setup wizard some files were missing. I think someone should look into this. And, like you asked me to check the `storm.css` file, it wasn't there when I installed using the setup wizard but was there when I installed via github.
